app/models/notifications/queries/IleQueryResponseExchangeData.scala

```diff
@@ -41,6 +42,8 @@ object IleQueryResponseExchangeData {
     childMucrs: Seq[MucrInfo] = Seq.empty
   ) extends IleQueryResponseExchangeData {
     override val typ = IleQueryResponseExchangeType.SuccessfulResponseExchange
+
+    val sortedChildrenUcrs: Seq[UcrInfo] = (childDucrs ++ childMucrs).sortBy(_.entryStatus.flatMap(_.roe).getOrElse(ROECode.UnknownRoe))
```

Comment:
I don't think this is a good place to put it. Not under this name at least. The logic seems fine, but the name is very generic while the method is actually doing very specific type of sorting.
I'd rather put this sorting where it is used as it is a very specific case of sorting.

Reply:
I agree.   This should just be a method that returns the combine child ucr list.  The sorting should be done by the view.   Other users might have different sorting requirements.